netlify/patch-files.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+const { copyFile, writeFile } = require('fs/promises')
+
+;(async () => {
+  console.log('Patch intro.md to include netlify banner.')
+  await copyFile('netlify/intro.md', 'public/mock-backend/public/intro.md')
+  console.log('Copied intro.md')
+
+  console.log('Patch motd.txt to include privacy policy.')
+  await copyFile('netlify/motd.txt', 'public/mock-backend/public/motd.txt')
+  console.log('Copied motd.txt')
+
+  console.log('Patch version.json to include git hash')
+  await writeFile(
+    'src/version.json',
+    JSON.stringify({
+      version: `git-${process.env.GITHUB_SHA}`,
```

Comment:
We probably don't want the full hash, but only the first 8 chars of it.

Reply:
And to follow semver, the format should be:

0.0.0+aaaaaaaa

where a is the short commit SHA.